### ISSUE TITLE
Review kernel density estimators (in particular in visualization)

### ISSUE THREAD

**Issue by yannikschaelte** [ICB-DCM/pyABC#42]
The default kernel density estimators apparently sometimes over-smoothen. In particular the ``MultivariateNormalTransition`` used for visualization is unsuited for multi-model landscapes (see attached picture). The problem in this case could be solved by reducing the ``scaling`` factor.
![image](https://user-images.githubusercontent.com/31767307/45883332-0aa13c00-bdb1-11e8-9588-60d103092771.png).
At least, one should allow the user to specify a kde here.

**Comment by yannikschaelte** [comment]
Aside from allowing the user to specify a kde here, I suggest to use GridSearchCV by default (5-fold cross-validation, range 0.1  to 1.0?), with a MultivariateNormalTransition. Then the density estimate will maybe tend to be more rugged, but at least not over-simplyfying, which I consider the worse of the two extremes, as it can lead to a wrong post-analysis.

**Comment by JanHasenauer** [comment]
Yes, the oversmoothing is unfortunately a problem. The GridSearch is a good idea!
